force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -288,8 +345,14 @@ public with sharing class FlowControlLoopsCollections {
     public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
+        decimal totalValue = 0;
+        for (Integer i = 0; i <= moneyInWallet.size(); i++) {
+            if (moneyInWallet[i] > 40.00) {
```

Comment:
You want to check if the total value is greater then 40, not if the individual amount to add is